runner/run_test.go

```diff
@@ -305,6 +305,14 @@ func (s *runTestSuite) TestRunTests_Run() {
 		s.Equal(res.Stats.TotalFailed(), 0, "failed to exclude test")
 	})
 
+	s.Run("count test tagged with `tag-1`", func() {
+		res, err := Run(s.cfg, s.ftwTests, RunnerConfig{
+			Tag: regexp.MustCompile("tag-1"),
+		}, s.out)
+		s.Require().NoError(err)
+		s.Equal(res.Stats.TotalFailed(), 0, "failed to incorporate tagged test")
```

Comment:
You need to also check the succeeded tests to make sure the tag matched. For that you need to update the test definitions at the beginning of the file.

The other tests are technically not entirely correct, as they should also check the succeeded and skipped counts. It would be nice if you would update the other tests as well (not required though).

Reply:
Sure, I can address the other tests but I will do it in a separate MR, right?

Reply:
That would be great.